packages/suite/src/components/suite/TransactionsGraph/components/SkeletonTransactionsGraph/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useLoadingSkeleton } from '@suite-hooks';
+import { SkeletonRectangle, Stack, Spread } from '@suite-components/Skeleton';
+
+const SkeletonWrapper = styled.div<{ animate?: boolean }>`
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    align-self: flex-end;
+    margin: 20px;
+`;
+
+const SkeletonBar = (props: React.ComponentProps<typeof SkeletonRectangle>) => (
+    <SkeletonRectangle borderRadius="4px 4px 0px 0px" {...props} />
+);
+
+const SkeletonTransactionsGraph = ({ animate, ...rest }: { animate?: boolean }) => {
+    const { shouldAnimate } = useLoadingSkeleton();
+
+    return (
+        <SkeletonWrapper animate={animate ?? shouldAnimate} {...rest}>
```

Comment:
just a UX suggestion - maybe we could randomize the bars so the user doesn't get bored soon

Reply:
hehe interesting idea, right after the release of final final v2 😁 
I would actually liked if we took some clue from games, which always had pretty long loading screens and they use it as a place to show hints or teach the player something interesting while waiting